helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -102,32 +102,24 @@ spec:
         - name: DD_CELERY_BROKER_PASSWORD
           valueFrom:
             secretKeyRef:
-            {{- if eq .Values.celery.broker "rabbitmq" }}
-            {{- if .Values.rabbitmq.enabled }}
-              name: defectdojo-{{ .Values.celery.broker }}-specific
-              key: {{ .Values.celery.broker }}-password
-            {{- else }}
-              name: {{ .Values.rabbitmq.existingPasswordSecret }}
-              key: {{ .Values.rabbitmq.secretPasswordKey }}  
-            {{- end }}            
-            {{- else if eq .Values.celery.broker "redis" }}
-            {{- if .Values.redis.enabled }}
-              name: defectdojo-{{ .Values.celery.broker }}-specific
-              key: {{ .Values.celery.broker }}-password
-            {{- else }}
-              name: {{ .Values.redis.existingSecret }}
-              key: {{ .Values.redis.secretKey }}
-            {{- end }}  
-            {{- end }}
+              {{- if eq .Values.celery.broker "rabbitmq" }}
+              name: {{ .Values.rabbitmq.existingPasswordSecret | default "defectdojo-rabbitmq-specific" }}
+              key: {{ .Values.rabbitmq.secretPasswordKey | default "rabbitmq-password" }}  
+              {{- end -}}
+              {{- if eq .Values.celery.broker "redis" }}
+              name: {{ .Values.redis.existingSecret | default "defectdojo-redis-specific" }}
```

Comment:
The same is for redis

Reply:
[Fixed](https://github.com/DefectDojo/django-DefectDojo/pull/6054/commits/11a4447cf457d9c791ee2251119a6a862f14c9fe).